CHANGELOG.md

```diff
@@ -9,7 +9,28 @@ _With the release of `v3.0.0`, we're introducing a new changelog format in an at
 
 _The old changelog can be found in the `release-2.6` branch_
 
-# Changes Since v3.0.1
+# Changes Since v3.0.2
+
+# v3.0.2 - []
+
+  - Added build requirements to RPM spec file for easier RPM building
```

Comment:
`Made minor improvements to RPM packaging and added basic support for alpine packaging`